guides/contribution.md

```diff
@@ -0,0 +1,119 @@
+# Contribution guide
+
+Welcome to the Mística Design System contribution guide for external teams.
+
+As an external team, you have the opportunity to contribute to the Mística Design System by sharing your ideas and expertise.
+Whether you are a designer, developer, or product manager, your contributions can help us improve the system and make it more useful for a wider range of users.
+
+This guide is designed to help you understand how you can contribute to Mística, what kind of contributions we are looking for, and how to make sure your contributions are consistent with our way of working.
+
+We encourage you to read this guide carefully and get in touch with us if you have any questions or feedback.
+
+## Index
+
+- [Proposal](#proposals)
+- [Creation of an issue based on the proposal](#issues)
+- [Updating or creating the specs needed](#specifications)
+- [Development and final review](#from-design-to-development)
+
```

Comment:
:robot: OpenAI

The index provides a clear outline of the topics covered in the guide. However, it would be helpful to provide a brief description of each section.
```suggestion
## Index

- [Proposal](#proposals): Learn how to submit a proposal for a new component or feature.
- [Creation of an issue based on the proposal](#issues): Understand how to create an issue based on an approved proposal.
- [Updating or creating the specs needed](#specifications): Find out how to update or create specifications for components or features.
- [Development and final review](#from-design-to-development): Discover the steps involved in moving from design to development.
```

<!-- This is an auto-generated comment by OpenAI -->